Dockerfile

```diff
@@ -66,11 +66,9 @@ EXPOSE 8000
 EXPOSE 5901
 
 RUN echo "mycontainer" > /etc/hostname
-RUN echo "127.0.0.1	localhost" > /etc/hosts
-RUN echo "127.0.0.1	mycontainer" >> /etc/hosts
 
-RUN addgroup --gid $GROUP_ID enhydris
-RUN adduser --uid $USER_ID --gid $GROUP_ID --disabled-password --gecos "" enhydris
+RUN groupadd -o --gid $GROUP_ID enhydris
+RUN adduser --uid $USER_ID --gid $GROUP_ID --gecos "" enhydris
```

Comment:
Without `--disabled-password`, doesn't this stop to ask for a password?

Reply:
No, it doesn't. I think the default behavior is to disable the password unless you set it with the `-p` flag. And when building the docker container, I was constantly getting an error that says the `--disabled-password` is unknown. Could it be that the `debian:bullseye-slim` image has a different version of the command?